toolkit/templates/summary-table.html

```diff
@@ -98,6 +98,12 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
   {% endcall %}
 {%- endmacro %}
 
+{% macro external_link(link_title, link) -%}
+  {% call field() %}
+    <a href="{{ link }}" rel="external">{{ link_title }}</a>
```

Comment:
Should we have a `nofollow` here in case it ends up being used by the public pages later?